flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/org/apache/flink/cdc/connectors/values/sink/ValuesDataSinkOptions.java

```diff
@@ -35,4 +35,10 @@ public class ValuesDataSinkOptions {
                     .booleanType()
                     .defaultValue(true)
                     .withDescription("True if the Event should be print to console.");
+
+    public static final ConfigOption<String> SINK_TYPE =
+            ConfigOptions.key("sinkType")
```

Comment:
What about `sink.api`? We use the word "type" in YAML for specifying the category of connector, such as "mysql", "doris", "values", so I'm concerned about the potential conflict.

Reply:
It sounds better. Just do it.